CHANGELOG.md

```diff
@@ -5,171 +5,121 @@
 
 This is the extensive list of all changes.
 
-Most noteable changes:
+### New features
 
+The changes mentioned here are discussed in detail in the [release highlights](https://medium.com/p/c1fbc08008da/), or were simply updated in the docs.
+
+* MobX 4 introduces separation between the production and non production build. The production build strips most typechecks, resulting in a faster and smaller build. Make sure to substitute process.env.NODE_ENV = "production" in your build process! If you are using MobX in a react project, you most probably already have set this up. Otherwise, they idea is explained [here](https://reactjs.org/docs/add-react-to-an-existing-app.html).
+* Observable maps are now backed by real ES6 Maps. This means that any value can be used as key now, not just strings and numbers.
+* Introduced `decorate(thing, decorators)` to decorate classes or object without needing decorator syntax.
```

Comment:
Any reason why this isn't in utils?

Reply:
Yes, this makes it possible to use MobX in the same when when using decorator syntax, plain objects, prototypes through `Object.create` etc.